deployments/ansible/roles/collector/handlers/main.yml

```diff
@@ -7,28 +7,38 @@
   listen:
     - restart splunk-otel-collector
     - restart td-agent
+  when:
+    - skip_restart | default("false") == "false"
```

Comment:
nit, vars that require negatives for enabling actions can be confusing where `start_service` w/ true default is clearer and doesn't center the restarting detail imo.